features/google/src/model/importers/access-log-importer.js

```diff
@@ -0,0 +1,45 @@
+import AccessLogEntry from "../entities/access-log-entry";
+
+const accessLogRegex = /\/Access Log Activity\/.*\.csv$/;
+
+class AccessLogParser {
+    constructor() {}
+
+    _csvToJson(csvText) {
```

Comment:
In terms of size this file can get to be around 30-40MB. We could later on look into something stream on based.

Reply:
But not at this point, I guess. It should happen at the import phase. Also, we're importing the zip into memory, and decompressing there; unless we want to put it back into a file I'm not sure that's practical at all.